### PR TITLE
8.patch/api/reviews/review

### PR DESCRIPTION
**Pull request by JoshFost**
tests 2 200 statues for different vote increases. and a 400 for if there is no inc votes and a 404 for for not existent review_ud

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -265,15 +276,68 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+    const comment = { username: "mallionaire", body: "test comment" };
+    return request(app)
+      .patch("/api/reviews/999")
+      .send(comment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Review not found");
+      });
+  });
 });
```
Comment (emilyb93) [inline __tests__/app.test.js]: I think we need another one or two tests here but this is looking great so far;
- Trying to send a valid patch to an id that isnt a number `/api/reviews/bananas/
- Sending a patch request body with an incorrect key
- Sending a patch request body with an incorrect value data type; `{inc_votes: 'one'}'

File: __tests__/app.test.js [hunk 2]
```diff
@@ -265,15 +276,68 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+      .send({})
       .expect(400)
       .then((response) => {
-        console.log(response);
+        console.log(response.body);
         expect(response.body.msg).toBe("Bad Request");
       });
   });
+  it("returns 404 error when review_id is not found", () => {
+    const comment = { username: "mallionaire", body: "test comment" };
```
Comment (emilyb93) [inline __tests__/app.test.js]: You are sending a comment body here when i think this means to be a patch body with inc_votes

**Review by emilyb93 — CHANGES_REQUESTED** [review]
see comments :)

**Review by johnhutcheon — APPROVED** [review]
Great stuff Josh, merge away! 😃

**Pull request merged**